packages/cli/src/bin/create-dev-serve.ts

```diff
@@ -84,7 +85,12 @@ export const createDevServer = async (options: {
 
     spinner.info('💾 application entrypoint', formatPath(String(entry)));
 
-    const devPortalPath = resolveRelativePath('public');
+    let devPortalPath: string;
+    if (options.devPortalPath) {
+        devPortalPath = resolveRelativePath(options.devPortalPath);
+    } else {
+        devPortalPath = resolveRelativePath('public');
+    }
```

Comment:
not too found of mutables

```suggestion
    const devPortalPath = resolveRelativePath(options.devPortalPath ?? 'public');
```

> __WARNING__ `resolveRelativePath` resolves relative to the import path of script aka bin folder of CLI

```ts
const resolveRelativePath = (path: string) => fileURLToPath(new URL(path, import.meta.url));
```

i guess the end user want is a relative path from `process.cwd`

i think that the resolving and fallback should maybe be in `main.app.ts`

https://github.com/equinor/fusion-framework/blob/c4e384152765d86df5093b35355d0c4b0bcfee43/packages/cli/src/bin/main.app.ts#L45-L58

Reply:
i guess during testing that you used an absolute path for `devPortalPath`?

Reply:
Yes, I used an absolute path; trying to make it relative programmatically was a bit of a pain when testing (due to npm link).